03-rnaseq/dimension-reduction_rnaseq_02_umap.Rmd

```diff
@@ -273,6 +264,26 @@ metadata <- metadata %>%
   )
 ```
 
+## Define a minimum counts cutoff
+
+We want to filter out the genes that have not been expressed or that have low expression counts because we want to remove any possible noise from our data before we normalize the data and create our heatmap.
```

Comment:
```suggestion
We want to filter out the genes that have not been expressed or that have low expression counts since these genes are likely to add noise rather than useful signal to our analysis.
```